lib/v2/screens/settings/settings_screen.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/i18n/settings.i18n.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
+import 'package:seeds/v2/screens/settings/components/settings_card.dart';
+
+const List<String> skillsAndInterest = ['Weaver', 'Facilitador', 'SharingXP'];
```

Comment:
leave a TODO: this is mock data and will be removed soon

Reply:
i dont see the todo yet?

Reply:
Strange I remember wrote that